client/reader/user-stream/index.tsx

```diff
@@ -87,6 +107,12 @@ const UserStream = ( { streamKey, userId }: UserStreamProps ) => {
 			<div className="user-stream__content">{ renderContent() }</div>
 		</div>
 	);
-};
+}
 
-export default UserStream;
+export default connect(
+	( state: AppState, ownProps: UserStreamProps ) => ( {
+		user: state.reader.users.items[ ownProps.userId ],
+		isLoading: state.reader.users.requesting?.[ ownProps.userId ] ?? false,
+	} ),
```

Comment:
This looks really ugly to me and I haven't found similar-looking `mapStateToProps` implementations in the codebase yet 🧐 Would appreciate any suggestions!